=== Suite started at 2024-11-26 10:00:00
=== Starting test, 13 test cases
=cases         13
=user          otptest
=host          ac22cca2be94
=hosts         ac22cca2be94 
=emulator_vsn  15.1.2
=emulator      beam
=otp_release   27
=started       2024-11-26 10:00:00
=case          cfg_error_9_SUITE:init_per_suite
=logfile       cfg_error_9_suite.init_per_suite.html
=started       2024-11-26 10:00:00
=ended         2024-11-26 10:00:00
=result        ok
=elapsed       0.000000s
=case          cfg_error_9_SUITE:tc1
=logfile       cfg_error_9_suite.tc1.html
=started       2024-11-26 10:00:00
=== ERROR! init_per_testcase crashed!
	Location: [{cfg_error_9_SUITE,init_per_testcase,76},
                   {test_server,do_init_per_testcase,1564},
                   {test_server,run_test_case_eval1,1265},
                   {test_server,run_test_case_eval,1235}]
	Reason: {tc1_should_be_skipped,
                    [{cfg_error_9_SUITE,init_per_testcase,2,
                         [{file,"cfg_error_9_SUITE.erl"},{line,76}]},
                     {test_server,do_init_per_testcase,2,
                         [{file,"test_server.erl"},{line,1564}]},
                     {test_server,run_test_case_eval1,6,
                         [{file,"test_server.erl"},{line,1265}]},
                     {test_server,run_test_case_eval,9,
                         [{file,"test_server.erl"},{line,1235}]}]}

=ended         2024-11-26 10:00:00
=result        auto_skipped: {failed,
                              {cfg_error_9_SUITE,init_per_testcase,
                               {tc1_should_be_skipped,
                                [{cfg_error_9_SUITE,init_per_testcase,2,
                                  [{file,"cfg_error_9_SUITE.erl"},{line,76}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1564}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,1265}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,1235}]}]}}}
=elapsed       0.000000s
=== *** SKIPPED test case 1 of 13 ***
=case          cfg_error_9_SUITE:tc2
=logfile       cfg_error_9_suite.tc2.html
=started       2024-11-26 10:00:00
=== ERROR! cfg_error_9_SUITE:init_per_testcase(tc2, [{watchdog,<0.266.0>},
                                                 {tc_logfile,
                                                  "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/cfg_error_9_suite.tc2.html"},
                                                 {tc_group_properties,[]},
                                                 {tc_group_path,[]},
                                                 {data_dir,
                                                  "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/cfg_error_9_SUITE_data/"},
                                                 {priv_dir,
                                                  "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/log_private/"}]) failed!
	Reason: {timetrap_timeout,2000}

=ended         2024-11-26 10:00:02
=result        auto_skipped: {failed,
                                 {cfg_error_9_SUITE,init_per_testcase,
                                     {timetrap_timeout,2000}}}
=elapsed       2.000000s
=== *** SKIPPED test case 2 of 13 ***
=case          cfg_error_9_SUITE:tc3
=logfile       cfg_error_9_suite.tc3.html
=started       2024-11-26 10:00:02
=== ERROR! init_per_testcase crashed!
	Location: [{cfg_error_9_SUITE,init_per_testcase,83},
                   {test_server,do_init_per_testcase,1564},
                   {test_server,run_test_case_eval1,1265},
                   {test_server,run_test_case_eval,1235}]
	Reason: {{badmatch,undefined},
                 [{cfg_error_9_SUITE,init_per_testcase,2,
                                     [{file,"cfg_error_9_SUITE.erl"},
                                      {line,83}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1564}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,1265}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,1235}]}]}

=ended         2024-11-26 10:00:02
=result        auto_skipped: {failed,
                              {cfg_error_9_SUITE,init_per_testcase,
                               {{badmatch,undefined},
                                [{cfg_error_9_SUITE,init_per_testcase,2,
                                  [{file,"cfg_error_9_SUITE.erl"},{line,83}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1564}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,1265}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,1235}]}]}}}
=elapsed       0.000000s
=== *** SKIPPED test case 3 of 13 ***
=case          cfg_error_9_SUITE:tc4
=logfile       cfg_error_9_suite.tc4.html
=started       2024-11-26 10:00:02
=== ERROR! init_per_testcase did not return a Config list.

=ended         2024-11-26 10:00:02
=result        auto_skipped: {failed,
                                 {cfg_error_9_SUITE,init_per_testcase,
                                     bad_return}}
=elapsed       0.000000s
=== *** SKIPPED test case 4 of 13 ***
=case          cfg_error_9_SUITE:tc5
=logfile       cfg_error_9_suite.tc5.html
=started       2024-11-26 10:00:02
=ended         2024-11-26 10:00:02
=result        failed: fail_this_testcase, [{cfg_error_9_SUITE,tc5}]
=elapsed       0.000000s
=== *** FAILED test case 5 of 13 ***
=case          cfg_error_9_SUITE:tc6
=logfile       cfg_error_9_suite.tc6.html
=started       2024-11-26 10:00:02
=ended         2024-11-26 10:00:02
=result        failed: fail_this_testcase, [{cfg_error_9_SUITE,tc6}]
=elapsed       0.000008s
=== *** FAILED test case 6 of 13 ***
=case          cfg_error_9_SUITE:tc7
=logfile       cfg_error_9_suite.tc7.html
=started       2024-11-26 10:00:02
=ended         2024-11-26 10:00:02
=result        failed: tc7_should_be_failed, [{cfg_error_9_SUITE,tc7}]
=elapsed       0.000000s
=== *** FAILED test case 7 of 13 ***
=case          cfg_error_9_SUITE:tc11
=logfile       cfg_error_9_suite.tc11.html
=started       2024-11-26 10:00:02
=== WARNING: end_per_testcase crashed!
Reason: warning_should_be_printed
Line: [{cfg_error_9_SUITE,end_per_testcase,100},
       {test_server,do_end_per_testcase,1638},
       {test_server,run_test_case_eval1,1346},
       {test_server,run_test_case_eval,1235}]

=ended         2024-11-26 10:00:02
=result        ok
=elapsed       0.000000s
=case          cfg_error_9_SUITE:tc12
=logfile       cfg_error_9_suite.tc12.html
=started       2024-11-26 10:00:02
=== WARNING! cfg_error_9_SUITE:end_per_testcase(tc12, [{tc_status,ok},
                                                   {watchdog,<0.281.0>},
                                                   {tc_logfile,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/cfg_error_9_suite.tc12.html"},
                                                   {tc_group_properties,[]},
                                                   {tc_group_path,[]},
                                                   {data_dir,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/cfg_error_9_SUITE_data/"},
                                                   {priv_dir,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/log_private/"}]) failed!
	Reason: timetrap timeout after 2000 ms!

=ended         2024-11-26 10:00:04
=result        ok
=elapsed       2.000000s
=case          cfg_error_9_SUITE:tc13
=logfile       cfg_error_9_suite.tc13.html
=started       2024-11-26 10:00:04
=== WARNING: end_per_testcase crashed!
Reason: {badmatch,undefined}
Line: [{cfg_error_9_SUITE,end_per_testcase,106},
       {test_server,do_end_per_testcase,1638},
       {test_server,run_test_case_eval1,1346},
       {test_server,run_test_case_eval,1235}]

=ended         2024-11-26 10:00:04
=result        ok
=elapsed       0.000000s
=case          cfg_error_9_SUITE:tc14
=logfile       cfg_error_9_suite.tc14.html
=started       2024-11-26 10:00:04
=ended         2024-11-26 10:00:04
=result        failed: tc14_should_be_failed, [{cfg_error_9_SUITE,tc14}]
=elapsed       0.000006s
=== *** FAILED test case 11 of 13 ***
=case          cfg_error_9_SUITE:tc15
=logfile       cfg_error_9_suite.tc15.html
=started       2024-11-26 10:00:04
=== WARNING: end_per_testcase crashed!
Reason: kaboom
Line: [{cfg_error_9_SUITE,end_per_testcase,111},
       {test_server,do_end_per_testcase,1638},
       {test_server,run_test_case_eval1,1346},
       {test_server,run_test_case_eval,1235}]

=ended         2024-11-26 10:00:04
=result        failed: this_error_must_show, [{cfg_error_9_SUITE,tc15,191},
                                              {test_server,ts_tc,1794},
                                              {test_server,
                                               run_test_case_eval1,1303},
                                              {test_server,
                                               run_test_case_eval,1235}]
=elapsed       0.000002s
=== *** FAILED test case 12 of 13 ***
=case          cfg_error_9_SUITE:tc16
=logfile       cfg_error_9_suite.tc16.html
=started       2024-11-26 10:00:04
=== WARNING! cfg_error_9_SUITE:end_per_testcase(tc16, [{tc_status,
                                                    {failed,
                                                     this_error_must_show}},
                                                   {tc_fail_loc,
                                                    [{cfg_error_9_SUITE,tc16,
                                                      193},
                                                     {test_server,ts_tc,1794},
                                                     {test_server,
                                                      run_test_case_eval1,
                                                      1303},
                                                     {test_server,
                                                      run_test_case_eval,
                                                      1235}]},
                                                   {watchdog,<0.290.0>},
                                                   {tc_logfile,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/cfg_error_9_suite.tc16.html"},
                                                   {tc_group_properties,[]},
                                                   {tc_group_path,[]},
                                                   {data_dir,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/cfg_error_9_SUITE_data/"},
                                                   {priv_dir,
                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@ac22cca2be94.2024-11-26_09.55.59/make_test_dir.common_test_test.logs/run.2024-11-26_09.56.02/log_private/ct_run.ct@ac22cca2be94.2024-11-26_09.59.40/ct_error_SUITE_data.error.cfg_error_9_SUITE.logs/run.2024-11-26_10.00.00/log_private/"}]) failed!
	Reason: timetrap timeout after 2000 ms!

=ended         2024-11-26 10:00:06
=result        failed: this_error_must_show, [{cfg_error_9_SUITE,tc16,193},
                                              {test_server,ts_tc,1794},
                                              {test_server,
                                               run_test_case_eval1,1303},
                                              {test_server,
                                               run_test_case_eval,1235}]
=elapsed       2.000000s
=== *** FAILED test case 13 of 13 ***
=case          cfg_error_9_SUITE:end_per_suite
=logfile       cfg_error_9_suite.end_per_suite.html
=started       2024-11-26 10:00:06
=ended         2024-11-26 10:00:06
=result        ok
=elapsed       0.000000s
=group_time    6.435s
=== TEST COMPLETE, 3 ok, 6 failed, 4 skipped of 13 test cases

=finished      2024-11-26 10:00:06
=failed        6
=successful    3
=user_skipped  0
=auto_skipped  4
=elapsed_time  6496837
